Persephone/Controllers/AlbumViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AlbumViewController.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/08.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+
+class AlbumViewController: NSViewController,
+                           NSCollectionViewDataSource,
+                           NSCollectionViewDelegate,
+                           NSCollectionViewDelegateFlowLayout {
```

Comment:
I might separate the `NSCollectionViewDelegateFlowLayout` into its' own object as well, since there's usually some gnarly arithmetic in its implementation.

Reply:
So I've spent some time trying to do this. I ended up subclassing `NSCollectionViewFlowLayout` and attempting to override the appropriate methods to get what I want, but it was starting to get complicated! Is there a simple way to split this stuff out?

Reply:
Ok, I figured this out over the weekend and it actually turned out to be much easier than I first thought. Update in PR #6